core/src/main/java/nl/nn/adapterframework/ldap/LdapChallengePipe.java

```diff
@@ -147,7 +148,7 @@ public PipeRunResult doPipe(Message msg, PipeLineSession pls) throws PipeRunExce
 			return new PipeRunResult(findForward("invalid"), msg);
 		}
 						
-		return new PipeRunResult(findForward("success"), msg);
```

Comment:
isn't this the getSuccessForward() ?

Reply:
no, LdapChallengPipe is not based on FixedForwardPipe (I think for no good reason)